### PR TITLE
Use clonerefs on all prow jobs

### PR DESCRIPTION
**Pull request by bbguimaraes**
@stevekuznetsov

### CODE REVIEW

File: sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-parent: 'common/test_cases/origin.yml'
+parent: 'common/test_cases/origin_minimal.yml'
 overrides:
   junit_analysis: False
   timer: 'H H * * *'
```
Comment (bbguimaraes) [inline sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml]: Is this timer still needed?

Comment (stevekuznetsov) [reply sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml]: Do we have it as a postsubmit in `openshift/release`?

Comment (bbguimaraes) [reply sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml]: Yes, these jobs are all pre/post submits in their respective repositories.

Comment (stevekuznetsov) [reply sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml]: OK, then no timer needed here any longer

Comment (bbguimaraes) [reply sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml]: Removed.

**Pull request merged**